adapter/src/dbt/adapters/fal_experimental/adapter.py

```diff
@@ -71,8 +76,32 @@ def run_in_environment_with_adapter(
     else:
         deps = get_default_pip_dependencies()
         stage = VirtualPythonEnvironment(deps)
+        fal_scripts_path = _get_fal_scripts_path(config)
 
-        with PythonIPC(environment, environment.create(), extra_inheritance_paths=[stage.create()]) as connection:
+        with PythonIPC(environment, environment.create(), extra_inheritance_paths=[fal_scripts_path, stage.create()]) as connection:
             execute_model = partial(_isolated_runner, code, config, manifest, macro_manifest)
             result = connection.run(execute_model)
             return result
+
+@contextmanager
+def extra_path(path: str):
+    import sys
+    sys.path.append(path)
+    try:
+        yield
+    finally:
+        sys.path.remove(path)
+
+def _get_fal_scripts_path(config: RuntimeConfig):
+    import pathlib
+    cwp = pathlib.Path.cwd()
+
+    fal_scripts_path = config.project_root
+
+    if hasattr(config, 'vars'):
+        fal_scripts_path: str = config.vars.to_dict().get(FAL_SCRIPTS_PATH_VAR_NAME, fal_scripts_path)  # type: ignore
+
+    if hasattr(config, 'cli_vars'):
+        fal_scripts_path = config.cli_vars.get(FAL_SCRIPTS_PATH_VAR_NAME, fal_scripts_path)
+
+    return cwp / fal_scripts_path
```

Comment:
ATL: What does this path look like if fal_scripts_path is the project root directory?